### PR TITLE
Component-Bright-Password-Bar-Component

### PR DESCRIPTION
**Pull request by Brightlaz**
password

### CODE REVIEW

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Please add a visual recording of your components in the PR so we see it's output.

**Comment by Brightlaz** [comment]
> Please add a visual recording of your components in the PR so we see it's output.

I added a visual recording above ☝️

**Review by Remi-dee — APPROVED** [review]

File: Developers/app/components/onboardTechies/passwordBar.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    setHasLowerCase(/[a-z]/.test(value));
+    setHasSpecialChar(/[!@#$%^&*()_+{}\[\]:;<>,.?~\\-]/.test(value));
+    setHasUpperCase(/[A-Z]/.test(value));
+    setHasNumber(/[0-9]/.test(value));
+  };
+
+  const ruleText = "text-xs font-light leading-none tracking-tight";
+  return (
+    <div
+      className={`${lexend.className} relative w-full max-sm:px-3 sm:w-[428px] mx-auto pt-8 space-y-[14px]`}
```
Comment (annonymousauthority) [inline Developers/app/components/onboardTechies/passwordBar.js:39]: There's no need adding a width like I said

File: Developers/app/components/onboardTechies/passwordBar.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        className="py-2 px-4 focus:outline-none bg-white rounded-[7px] w-full m-auto shadow border border-gray-200 text-black text-lg font-light leading-normal tracking-tight"
+      />
+      {isFocused && (
+        <div
+          className={`relative space-y-3 opacity-${isFocused ? "100" : "0"}`}
+        >
+          <p className="text-black text-xs font-light leading-none tracking-tight text-left">
+            Password should contain
+          </p>
+          <div className="flex justify-between sm:pr-[41px]">
```
Comment (annonymousauthority) [inline Developers/app/components/onboardTechies/passwordBar.js:63]: Dangerous `pr-41px` is probably too high.

But if you can't change it. Just quickly make a PR

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Quick Review

File: Developers/app/components/onboardTechies/passwordBar.js (hunk 2 above)
Comment (Brightlaz) [reply Developers/app/components/onboardTechies/passwordBar.js:63]: I calculated it from directly from the design file and have checked it across all screens and it's just right ✅

**Review by annonymousauthority — APPROVED** [review]
I will make changes after merging

**Pull request merged**